Source/encrypt.cpp

```diff
@@ -16,7 +16,7 @@ void Decrypt(DWORD *castBlock, DWORD size, DWORD key)
 
 	seed = 0xEEEEEEEE;
 	for (i = 0; i < (size >> 2); i++) {
-		DWORD t = SwapLE32(*castBlock);
+		DWORD t = SDL_SwapLE32(*castBlock);

```

Comment:
This looks unrelated to moving macros?

Reply:
it removed an old dev macro that doesn't make sens in devx.